src/zzz_od/gui/app.py

```
@@ -223,6 +231,26 @@ def _after_app_launch(self):
             self._check_first_run()
 
+        def _check_game_running(self) -> None:
+            """定时检测游戏进程是否运行，运行中则累计分钟数"""
+            game_path = self.ctx.game_account_config.game_path
+            if not game_path:
+                return
+            exe_name = os.path.basename(game_path)
+            if not exe_name:
+                return
+            try:
+                result = subprocess.run(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)